fabtests/ubertest/config.c

```diff
@@ -839,6 +865,19 @@ void fts_cur_info(struct ft_series *series, struct ft_info *info)
 			info->mode |= set->mode[i++];
 	}
 
+	if (set->cq_format[0]) {
+		info->cq_format = set->cq_format[series->cur_cq_format];
+	} else {
+		if (info->test_class & FI_TAGGED)
+			info->cq_format = FI_CQ_FORMAT_TAGGED;
+		else if (info->test_class & FI_MSG)
+			info->cq_format = FI_CQ_FORMAT_MSG;
+		else if (info->msg_flags & FI_REMOTE_CQ_DATA)
```

Comment:
Is this check correct?  What is the test_class if cq data is specified?  I.e. is the if check immediately above this going to trigger, skipping this check?